src/main/java/bio/terra/service/snapshot/SnapshotTableDao.java

```diff
@@ -93,42 +111,59 @@ private void createColumns(UUID tableId, Collection<Column> columns) {
   }
 
   public List<SnapshotTable> retrieveTables(UUID parentId) {
```

Comment:
If this is the snapshot ID maybe the change the parameter name too?
```suggestion
  public List<SnapshotTable> retrieveTables(UUID snapshotId) {
```